src/ApplicationInsights.ServiceFabric.Native/Net45/IWrappingClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.ServiceFabric.Remoting.Activities
+{
+    using Microsoft.ServiceFabric.Services.Remoting.Client;
+
+    /// <summary>
+    /// Interface implemented by remoting clients to provide access to the inner client.
+    /// </summary>
+    internal interface IWrappingClient
```

Comment:
It looks like FabricTransportServiceRemotingClientFactory.ReportOperationExceptionAsync() will perform a cast on the client that's passed in to make sure it's of type FabricTransportServiceRemotingClient. We need to keep the InnerClient property and access it from CorrelatingServiceRemotingClientFactory.